web/src/engine/websites/RuManhua.ts

```diff
@@ -0,0 +1,92 @@
+import { Tags } from '../Tags';
+import icon from './RuManhua.webp';
+import { Chapter, DecoratableMangaScraper, Manga, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchJSON } from '../platform/FetchProvider';
+
+type APIResponse<T> = {
+    data : T | string
```

Comment:
Is the union with `string` required?
I didn't see any business logic that processes the `data` field as string.

Reply:
there is no "undefined or null" here. In case the api response is empty, data is an empty string. Otherwise its type is T.

Perhaps i should replace string with "" ?